cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
That's being taken care of by https://github.com/rook/rook/pull/6424

Reply:
I just changed that because the linter is otherwise failing. I'm happy to wait for #6424

Reply:
Yes, it's a much more involved change than this. I've triggered the CI (it was incorrectly being skipped by jenkins) and expect that it will fail based on this. Is this change necessary to build with helm 3?

Reply:
As I said - the linter failed (the error is in the commit message). Not sure if it is really needed or just a linter thing.